server/service/transport.go

```diff
@@ -420,6 +420,11 @@ func hostListOptionsFromRequest(r *http.Request) (fleet.HostListOptions, error)
 		)
 	}
 
+	connectedToFleet := r.URL.Query().Has("connected_to_fleet")
```

Comment:
Ultimately a product call, but IMHO the URL query param ought to be `mdm_connected_to_fleet` or `connected_to_fleet_mdm` since a user encountering this in a URL doesn't have the benefit of the additional context clues available to a user encountering this inside the `host.mdm` response object.

Reply:
I asked something similar and in the end it was decided this is fine so I'm going to leave it as-is